.github/workflows/build.yml

```diff
@@ -24,22 +24,40 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.9", "3.10", "3.11", "3.12"]
+        uv-resolution: ["highest", "lowest"]
```

Comment:
Maybe we relegate this to less frequent running tests? e.g. only once per day, or once per week? Or do you think this is fast enough that it's worth running every time?